ckanext/switzerland/templates/base.html

```diff
@@ -46,6 +46,8 @@
   <!-- Piwik -->
   <script type="text/javascript">
     var _paq = _paq || [];
+    var customDimensionActionDatasetId = {{ piwik_config.custom_dimension_action_dataset_id or 0 }}
+    var customDimensionActionFormatId = {{ piwik_config.custom_dimension_action_format_id or 0 }}
```

Comment:
I'm not really sure if we should add these variables to the Piwik Snippet. I would create a new script block (which is loaded before app.js) and add them there.

Reply:
I don't know where to put it since the values get read from the ckan config and this variable is  set right before on line 45....These variables are piwik related after all and this block uses the config helper as well.

Reply:
I would do something like this:
```
{%- block scripts %}
  <script type='text/javascript' src='https://code.jquery.com/jquery-1.11.1.min.js?ver=1.11.1'></script>
  {% set piwik_config = h.get_piwik_config() %}
  <script type="text/javascript">
    var customDimensionActionDatasetId = {{ piwik_config.custom_dimension_action_dataset_id or 0 }}
    var customDimensionActionFormatId = {{ piwik_config.custom_dimension_action_format_id or 0 }}
  </script>
  <script type='text/javascript' src='/content/themes/wp-ogdch-theme/assets/js/app.min.js?ver=1.0.0'></script>
{% endblock -%}

{% block body_extras %}
  {% set piwik_config = h.get_piwik_config() %}
  <!-- Piwik -->
  <script type="text/javascript">
    var _paq = _paq || [];
    _paq.push(['trackPageView']);
...
```

Reply:
makes sense

Reply:
ok, done